cypress/integration/publication.spec.js

```diff
@@ -82,6 +82,19 @@ describe('Dataset Publication', () => {
                 .should('be.visible');
         });
 
+        it('should allow to fix obsolete imported model before publish dataset', () => {
+            menu.openAdvancedDrawer();
+            menu.goToAdminDashboard();
+            datasetImportPage.importDataset('dataset/simple.csv');
+            datasetImportPage.importModel('model/concat-obsolete.json');
+
+            cy.contains('Errors').click();
+            cy.contains('Operation UNKNOWN-OPERATION does not exist').click();
+
+            datasetImportPage.setOperationTypeInWizard('BOOLEAN');
+            datasetImportPage.publish();
```

Comment:
It's stange that this test end with a publish, and not with a testing instruction.
Is the publish needed ? 
And if it does, isn't it testable ?

Reply:
I can test it's "publishable", but publish will fail if it's not. So, it act like a test on "publishable".
You known what I mean?

Reply:
Forget it, it's ok